components/html-2-objects-compare.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * File:  html-2-objects-compare.js
+ *
+ * This component generates an HTML file that shows a deep dump of two Javascript objects side by side
+ * for easy visual comparison.
+ *
+ * Data coming into the "left" port will be shown in the left side column; data coming into the "right"
+ * port will be shown in the right side column of the HTML page.  The title will be taken from the IIP 
+ * constant configuration for each port in the form of {"title":"<column title>"}.
+ */
+ 
+var _ = require('underscore');
+var util = require('util');
+var fs = require('fs');
+var open = require('open');
+
+var jswrapper = require('../src/javascript-wrapper');
```

Comment:
Correct.  As with the merge-patient code in sprint 5, javascript-wrapper is part of a separate check in.  This is simply to add a clean, very simple standard component to our toolbox of components.   It should run on whatever javascript-wrapper we decide to go with.   There is one available on the demo branch that could be used temporarily.   

I will be checking in a version of javascript-wrapper, framework, vni, and Input state later today(hopefully) in a separate pull request.  I expect that is going to involve a whole lot more discussion and changes which is why I kept this separate.